Gems/AtomLyIntegration/AtomViewportDisplayInfo/Code/Source/AtomViewportDisplayInfoSystemComponent.cpp

```diff
@@ -413,6 +431,17 @@ namespace AZ::Render
             return value > upperLimit ? "inf" : AZStd::string::format(format, value);
         };
 
+#if defined (CARBONATED)
+        if (drawFpsOnly)
+        {
+            DrawLine(
+                AZStd::string::format(
+                    "FPS %s",
+                    ClampedFloatDisplay(averageFPS, "%.1f").c_str()),
+                AZ::Colors::Yellow);
+            return;
+        }
+#endif
```

Comment:
It'd be nice to have a min/max over last 5 seconds or whatever as well.  Not for this ticket, but future enhancement.

Reply:
^^^ FYI:
a) We could return back the full FPS output line "FPS %s [%s..%s], %sms/frame, avg over %.1fs", or may be its part "FPS %s [%s..%s]"
b) We could set r_fpsCalcInterval into 5 seconds instead 1 second as it is initialized by default

Reply:
Updated (as it is possible)